### ISSUE TITLE
Import ABC from collections.abc instead of collections for Python 3.9 compatibility.

### ISSUE THREAD

**Issue by tirkarthi** [python-cmd2/cmd2#851]
Import ABC from collections.abc instead of collections for Python 3.9 compatibility.

https://github.com/python-cmd2/cmd2/blob/e4bc87da34bb6fc645f54a4467dc7d0e32dfaa3b/cmd2/utils.py#L60

**Comment by kmvanbrunt** [comment]
@tirkarthi 
Thanks for the PR. We hadn't tested cmd2 with Python 3.9 yet.
The changes you made won't fix the issue with cliff though.
It's [requirements.txt](https://opendev.org/openstack/cliff/src/branch/master/requirements.txt) says: (cmd2>=0.8.0,!=0.8.3,<0.9.0 # MIT)

cmd2 versions from the 0.8 family have not been supported since August 2018. What is the possibility of cliff upgrading to use a newer cmd2 release?

@tleonhardt How do you feel about releasing a cmd2 0.8.10 version?

**Comment by tirkarthi** [comment]
Okay, I didn't look into cliff requirements. I came here from fedora's report for cliff where cmd2 was incompatible.

**Comment by tleonhardt** [comment]
I don't feel good about releasing another release in the 0.8 line, though I wouldn't object to doing it once and only once.  At this point that code is 1.5 years old however and we haven't used or tested it in that time.  `cmd2` has come so far from that point that I almost feel embarrassed releasing an update to that ancient code.  However, if it can provide a good customer experience for some users who are picking up `cmd2` as a transient dependency, then maybe that is alright. 

Regardless, the existing PR seems good for fixing a Python 3.9 compatibility problem.

**Comment by tleonhardt** [comment]
@tirkarthi Thank you for the PR.  I am going to merge it in since it does fix the Python 3.9 compatibility issue that it was intended to.

If you are concerned about `cliff` compatibility then you should probably create another issue since that is logically independent and would need a different solution.

**Comment by tirkarthi** [comment]
Got it, thanks @tleonhardt for the review.
